command/main.go

```diff
@@ -23,8 +47,85 @@ func Run(args []string) int {
 			args = []string{"version"}
 			break
 		}
+
+		if arg == "-no-color" {
```

Comment:
UX-ey thing. We should be nice and add `|| arg == "-nocolor"`. Even if we never advertise that, people will forget the dash and "auto-correcting them" will be a nice UX.